README.md

```diff
@@ -96,12 +96,16 @@ cat $HOME/.dispatch/config.json
     "port": <port>,
     "organization": "",
     "cookie": "",
-    "insecure": true,
```

Comment:
I agree with the below note. But don't we want insecure to be true for the laptop quick start where we are also skipping auth?

Reply:
I just posted a PR without seeing this.